package.json

```diff
@@ -29,13 +29,20 @@
     "lint": "run-p lint:*",
     "build-storybook": "storybook build",
     "start": "TS_NODE_PROJECT=.storybook/tsconfig.json storybook dev -p 7008",
-    "prepublishOnly": "npm run build"
+    "prepublishOnly": "npm run build",
+    "playwright:install": "playwright install --with-deps",
+    "playwright": "playwright test --config=playwright/playwright.config.ts",
+    "playwright:update": "npm run playwright -- -u",
+    "playwright:docker": "./scripts/playwright-docker.sh 'npm run playwright'",
+    "playwright:docker:update": "./scripts/playwright-docker.sh 'npm run playwright:update'",
+    "playwright:docker:clear-cache": "./scripts/playwright-docker.sh clear-cache"
   },
   "dependencies": {
     "react-transition-group": "^4.4.1",
     "tslib": "^2.4.0"
   },
   "devDependencies": {
+    "@andylacko/vite-svg-react-loader": "^3.0.16",
```

Comment:
"This plugin is deprecated" 👀
https://www.npmjs.com/package/@andylacko/vite-svg-react-loader#this-plugin-is-deprecated

Reply:
changed to another plugin vite-plugin-svgr